plugins/search/src/components/HomePageComponent/HomePageSearchBar.tsx

```diff
@@ -66,12 +67,14 @@ export const HomePageSearchBar = ({
   );
 
   return (
-    <SearchBarBase
-      className={className}
-      value={query}
-      onSubmit={handleSubmit}
-      onChange={handleChange}
-      {...props}
-    />
+    <SearchContextProvider>
```

Comment:
Right.. my bad! Thanks for opening the PR. I do think though that we actually want to keep SearchBarBase decoupled from the search context. So maybe we should move the track search component to inside the SearchBar component instead of inside SearchBarBase, what do you think?

Reply:
hm would that mean searches made from the `HomePageSearchBar` won't be tracked?

Reply:
or would `HomePageSearchBar` wrap `SearchBar` instead of `SearchBarBase` 🤔

Reply:
I'm also thinking there are situations where the homepage search bar is actually legitimately inheriting from an existing search context that we don't want to usurp.

Maybe the ideal solution would be to only wrap the underlying component in a search context provider if no search context is currently known/available?